### PR TITLE
Feature/detail

### LINKED ISSUE #29: Detail page
without functions related to comments

### PR DESCRIPTION
**Pull request by lyuboxin**
Resolved #29 create detail page.

### CODE REVIEW

**Review by zhangjiangqige — CHANGES_REQUESTED** [review]
should remove detail_.html (git rm)

File: homepage.html [hunk 1]
```diff
@@ -307,10 +306,13 @@ <h3 style='text-align: center'>
                 search_result[index].date_end +
                 "&nbsp&nbsp&nbsp" +
                 "</td>";
-              var home_id_search = room_information[index].house_id;
-              var detail_search = "<a href=\"detail.html?home_id="+home_id_search+"\">details</a>";
+              var home_id_search = search_result[index].house_id;
+              var detail_search = "<a href=\"detail_.html?home_id="+home_id_search+"\">details</a>";
```
Comment (xingweitian) [inline homepage.html]: do you need to change **detail_** to **detail**?

File: homepage.html [hunk 2]
```diff
@@ -371,7 +373,7 @@ <h3 style='text-align: center'>
                 "&nbsp&nbsp&nbsp" +
                 "</td>";
               var home_id = room_information[index].house_id;
-              var detail = "<a href=\"detail.html?home_id="+home_id+"\">details</a>";
+              var detail = "<a href=\"detail_.html?home_id="+home_id+"\">details</a>";
```
Comment (xingweitian) [inline homepage.html]: the same as above

File: homepage.html (hunk 1 above)
Comment (lyuboxin) [reply homepage.html]: I need to change href to detail too and I forgot. I will fix it soon

File: homepage.html (hunk 2 above)
Comment (lyuboxin) [reply homepage.html]: I will fix it

**Review by xingweitian — APPROVED** [review]

**Review by zhangjiangqige — APPROVED** [review]

**Pull request merged**